### ISSUE TITLE
mixpanel tracking SSL cert failure

### ISSUE THREAD

**Issue by tekumara** [sourcery-ai/sourcery-vscode#35]
```
Traceback (most recent call last):
  File "/Users/tekumara/.vscode/extensions/sourcery.sourcery-0.8.8/binaries/sourcery-0.8.8-mac/threading.py", line 926, in _bootstrap_inner
  File "/Users/tekumara/.vscode/extensions/sourcery.sourcery-0.8.8/binaries/sourcery-0.8.8-mac/sentry_sdk/integrations/threading.py", line 69, in run
  File "/Users/tekumara/.vscode/extensions/sourcery.sourcery-0.8.8/binaries/sourcery-0.8.8-mac/sentry_sdk/_compat.py", line 54, in reraise
  File "/Users/tekumara/.vscode/extensions/sourcery.sourcery-0.8.8/binaries/sourcery-0.8.8-mac/sentry_sdk/integrations/threading.py", line 67, in run
  File "/Users/tekumara/.vscode/extensions/sourcery.sourcery-0.8.8/binaries/sourcery-0.8.8-mac/mixpanel_async/async_buffered_consumer.py", line 37, in run
  File "/Users/tekumara/.vscode/extensions/sourcery.sourcery-0.8.8/binaries/sourcery-0.8.8-mac/mixpanel_async/async_buffered_consumer.py", line 291, in _sync_flush
  File "/Users/tekumara/.vscode/extensions/sourcery.sourcery-0.8.8/binaries/sourcery-0.8.8-mac/mixpanel/__init__.py", line 735, in _flush_endpoint
  File "<string>", line 3, in raise_from
mixpanel.MixpanelException: HTTPSConnectionPool(host='api-eu.mixpanel.com', port=443): Max retries exceeded with url: /track (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate in certificate chain (_ssl.c:1091)')))
```
```

**Comment by Hellebore** [comment]
Should be fixed in the next release.

**Comment by Hellebore** [comment]
Should be fixed with release of 0.9.0.
